client/app/ResultsPage/Map.jsx

```diff
@@ -1,26 +1,93 @@
 import React, { Component, PropTypes } from 'react';
 import {FancyBorder} from '../helpers.js';
 
-// INCOMPLETE
-  // has no interaction with user inputs or events
-const Map = props => (
-  <div>
-    <div className="row">
-      <h1 className="text-center">Results Map</h1>
-    </div>
-    <div className="row mapContainer">
-      <iframe
-        frameBorder="0"
-        src={`https://www.google.com/maps/embed/v1/directions?key=AIzaSyCBYpJtIs3LND6eRzsNbPldwx9cqHh3WOM&origin=${props.userQuery.startingLocation}&destination=${props.selectedActivities[props.selectedActivities - 1]}`}
-        allowFullScreen
-        className="map"
-      />
-
-    </div>
-  </div>
-
-  );
+class Map extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      //start: props.userQuery.startingLocation
```

Comment:
i think that the state needs to go in the requestPage component, if were planning on having a page after this for booking flights and finalizing things. is that something you want to do now or later?